app/src/Components/AddContact.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+const AddContact = ({ addContact }) => {
+  const onSubmit = (e) => {
+    const form = e.currentTarget;
+    const {
+      name: { value: name },
+      email: { value: email },
+      phone: { value: phone },
+      notes: { value: notes },
+      funfact: { value: funfact },
+    } = form.elements;
+
+    e.preventDefault();
+    addContact({
+      name,
+      email,
+      phone,
+      notes,
+      funfact,
+    });
+    form.reset();
+  };
+
+  return (
+    <>
+      <h2>Add a New Contact</h2>
+      <form onSubmit={onSubmit}>
+        <label htmlFor="name">Name: </label>
+        <input id="name" name="name" required />
+        <label htmlFor="email">Email: </label>
+        <input id="email" name="email" required />
```

Comment:
[fix] Don't forget your HTML input type to get some free validations and better mobile keyboard experience.